lib/rules/no-unused-vars.js

```diff
@@ -99,7 +99,6 @@ module.exports = {
          * Determines if a given variable is being exported from a module.
          * @param {Variable} variable - EScope variable object.
          * @returns {boolean} True if the variable is exported, false if not.
-         * @private
```

Comment:
ok-ok, also, i think it would the right move to do this everywhere else, how about creating a ticket about this?